Remotion/Data/DomainObjects.UnitTests/QueriesForStandardMapping.xml

```diff
@@ -138,30 +138,47 @@
     </statement>
   </query>
 
-  <query id="StoredProcedureQuery" type="collection">
+  <query id="StoredProcedureQuery" type="collection-readonly">
     <storageGroupType>Remotion.Data.DomainObjects.UnitTests::TestDomain.TestDomainAttribute</storageGroupType>
     <statement>rpf_testSPQuery</statement>
     <collectionType>Remotion.Data.DomainObjects.UnitTests.TestDomain.OrderCollection, Remotion.Data.DomainObjects.UnitTests</collectionType>
   </query>
 
-  <query id="StoredProcedureQueryWithParameter" type="collection">
+  <query id="StoredProcedureQueryWithParameter" type="collection-readonly">
     <storageGroupType>Remotion.Data.DomainObjects.UnitTests::TestDomain.TestDomainAttribute</storageGroupType>
     <statement>exec rpf_testSPQueryWithParameter @customerID</statement>
     <collectionType>Remotion.Data.DomainObjects.UnitTests.TestDomain.OrderCollection, Remotion.Data.DomainObjects.UnitTests</collectionType>
   </query>
 
-  <query id="FakeQuery" type="collection">
+  <query id="FakeQuery" type="collection-readonly">
     <storageGroupType>Remotion.Data.DomainObjects.UnitTests::TestDomain.TestDomainAttribute</storageGroupType>
     <statement>SELECT "Dummy"</statement>
     <collectionType>Remotion.Data.DomainObjects::DomainObjectCollection</collectionType>
   </query>
 
-  <query id="CustomQuery" type="custom">
+  <query id="CollectionQuery" type="collection-readonly">
+    <storageGroupType>Remotion.Data.DomainObjects.UnitTests::TestDomain.TestDomainAttribute</storageGroupType>
+    <statement>SELECT "Dummy"</statement>
+    <collectionType>Remotion.Data.DomainObjects::DomainObjectCollection</collectionType>
+  </query>
+
+  <query id="CollectionQueryReadWrite" type="collection-readwrite">
+    <storageGroupType>Remotion.Data.DomainObjects.UnitTests::TestDomain.TestDomainAttribute</storageGroupType>
+    <statement>SELECT "Dummy"</statement>
+    <collectionType>Remotion.Data.DomainObjects::DomainObjectCollection</collectionType>
+  </query>
+
+  <query id="CustomQuery" type="custom-readonly">
```

Comment:
```suggestion
  <query id="CustomQueryReadOnly" type="custom-readonly">
```
(the query existed before but it's also a complete and obvious test/dummy query. we should thus keep the name obvious to the intent. (this differes from a query that's already got a business-domain name)